ui/app/src/services/monitoring.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2007-2020 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { get } from '../utils/ajax';
+import { pluck } from 'rxjs/operators';
+import { Observable } from 'rxjs';
+import { StudioVersion } from '../models/Version';
+
+export function getStudioInfo(): Observable<StudioVersion> {
+  return get('/studio/api/2/monitoring/version').pipe(
```

Comment:
@jdrossl is this info available to use on the FTL context? 👆🏻

Reply:
not directly in freemarker, you would need to do it in groovy and pass it